src/Land/Model/land_tendencies.jl

```diff
@@ -9,19 +9,36 @@ eq_tends(pv::AbstractPrognosticVariable, m::LandModel, tt::Source) =
 ##### First order fluxes
 #####
 
-eq_tends(::PV, ::LandModel, ::Flux{FirstOrder}) where {PV} = ()
+eq_tends(pv::PV, land::LandModel, tt::Flux{FirstOrder}) where {PV} = (
+    eq_tends(pv, land.soil.heat, tt)...,
+    eq_tends(pv, land.soil.water, tt)...,
+    eq_tends(pv, land.surface, tt)...,
+)
+# can we replace these empty ones with a method for ::BalanceLaw?
```

Comment:
# can we replace these empty ones with a method for ::BalanceLaw?

Reply:
Yes, I think we could define a highly generic fallback. I think I didn't at first just to be careful, but since everything shows up in the table we should know if we're accidentally calling the fallback. Probably better to do this in a separate PR, though.

Reply:
ok! ill open up an issue so we dont forget